mongo-init.js

```diff
@@ -0,0 +1,65 @@
+/* eslint no-undef: "off" */
```

Comment:
I am assuming this script should only run when initializing the local dev instance right? Is there some way we can clearly delineate that?